plugins/funind/recdef.ml

```diff
@@ -1614,7 +1617,7 @@ let recursive_definition is_mes function_name rec_impls type_of_f r rec_arg_num
       tcc_lemma_constr
       is_mes functional_ref
       (EConstr.of_constr rec_arg_type)
-      (EConstr.of_constr relation) rec_arg_num
+      relation rec_arg_num
       term_id
       using_lemmas
       (List.length res_vars)
```

Comment:
What to do in `funind` would require the opinion of an expert. Maintaining the status quo as done here seems otherwise the reasonable thing to do.